ccdproc/image_collection.py

```diff
@@ -720,3 +738,9 @@ def data(self, do_not_scale_image_data=False, **kwd):
     data.__doc__ = _generator.__doc__.format(name='image',
                                              default_scaling='False',
                                              return_type='numpy.ndarray')
+
+    def ccddata(self, ccd_kwargs={}, **kwd):
```

Comment:
I know it is already inconsistent but there is `headers` and `hdus` and I think the plural does make sense here, maybe `ccds` as name?